src/params.js

```diff
@@ -57,7 +57,9 @@ const rootDir = getWorkspacesRoot(projectRoot);
 
 const rootPacakgeJson = JSON.parse(fs.readFileSync(path.join(rootDir, 'package.json'), 'utf-8'));
 
-const projectWorkspaces = JSON.parse(execSync('yarn workspaces --silent info', { cwd: rootDir }).toString());
+const projectWorkspaces = JSON.parse(
+  execSync('yarn workspaces --silent info', { cwd: rootDir, encoding: 'utf8' }).toString().replace(`[2K[1G`, ''),
```

Comment:
?

Reply:
yea. in different versions of yarn there's a different outputs... i use 1.22.10 and it generates a different json output...